subsys/net/lib/dhcpv4/dhcpv4.c

```diff
@@ -46,8 +46,12 @@ static struct net_mgmt_event_callback mgmt4_cb;
 
 #if defined(CONFIG_NET_DHCPV4_OPTION_CALLBACKS)
 static sys_slist_t option_callbacks;
+static int unique_types_in_callbacks;
 #endif
 
+static const uint8_t min_req_options[] = {DHCPV4_OPTIONS_SUBNET_MASK, DHCPV4_OPTIONS_ROUTER,
+				    DHCPV4_OPTIONS_DNS_SERVER};
```

Comment:
A small formatting suggestion
```suggestion
static const uint8_t min_req_options[] = {
	DHCPV4_OPTIONS_SUBNET_MASK, 
	DHCPV4_OPTIONS_ROUTER,
	DHCPV4_OPTIONS_DNS_SERVER
};
```